stream-applications-release-train/stream-applications-descriptor/pom.xml

```diff
@@ -15,12 +15,13 @@
     <packaging>jar</packaging>
     <properties>
         <uniqueVersion>false</uniqueVersion>
+        <groovy.version>3.0.23</groovy.version>
```

Comment:
This is very out-dated artifact: https://groovy.apache.org/download.html.
Good to see it is working with whatever Spring Integration does with Groovy!
But probably better to upgrade this artifact group to a newer version.